.travis.yml

```diff
@@ -10,10 +10,25 @@ matrix:
   include:
      - python: 3.6
        env:
+          - BUILD_DOCS=1
           - BUILD=1
           - PYDM_CHANNEL=pcds-tag
      - python: 3.6
        env: PYDM_CHANNEL=pydm-dev
+     - python: 3.7
+       dist: xenial
+       sudo: true
```

Comment:
I think `sudo` should be required on all, given the `apt-get` calls below.

Reply:
I have `sudo: true` at the top. Maybe this is just redundant